docs/docs/reference/release-notes/image-migration-guide.md

```diff
@@ -90,9 +90,9 @@ In addition to the syntax changes for using gatsby-plugin-image, there are also
 
 The `fluid` image type has been deprecated in favor of two alternatives.
 
-The first is an image type called `fullWidth`. This image is designed to be used for things like hero images and generates image sizes accordingly. Instead of passing something like `maxWidth`, it takes an array called `breakpoints` that will generate images designed for those screen sizes. Note that `fullWidth` images will expand to fit the width of their container, even if that width is larger than the source image.
+The first is an image type called `fullWidth`. This image is designed to be used for things like hero images and generates image sizes accordingly. Instead of passing something like `maxWidth`, it takes an array called `breakpoints` that will generate images designed for those screen sizes. In most cases will not need to provide these, as there are default values that work for most sites. Note that `fullWidth` images will expand to fit the width of their container, even if that width is larger than the source image.
```

Comment:
```suggestion
The first is an image type called `fullWidth`. This image is designed to be used for things like hero images and generates image sizes accordingly. Instead of passing something like `maxWidth`, it takes an array called `breakpoints` that will generate images designed for those screen sizes. In most cases you will not need to provide these, as there are default values that work for most sites. Note that `fullWidth` images will expand to fit the width of their container, even if that width is larger than the source image.
```
I think this was my typo!